packages/ui/src/services/breakpoints/plugin/create-service.ts

```diff
@@ -0,0 +1,138 @@
+import { App, ref, Ref, computed, watch } from 'vue'
+
+import { warn } from '../../utils'
+import { getGlobalProperty } from '../../../vuestic-plugin/utils'
+import { addOrUpdateStyleElement } from '../../dom-functions'
+
+import { GlobalConfig } from '../../global-config/types'
+import { ThresholdsKeys, BreakpointsConfig, BodyClass } from '../types'
+
+export const createBreakpointsConfigPlugin = (app: App) => {
+  const globalConfig: Ref<GlobalConfig> | undefined = getGlobalProperty(app, '$vaConfig')?.globalConfig
+  if (!globalConfig) {
+    warn('createBreakpointsConfigPlugin: globalConfig is not defined!')
+    return {}
+  }
+
+  const breakpointsConfig: BreakpointsConfig | undefined = globalConfig.value.breakpoints
```

Comment:
Looks like we are losing reactivity here. I know that breakpoints config shouldn't be reactive. We need to think how can we make runtime and non-runtime config. Non-runtime config later can be replaced with bundler plugin or nuxt plugin. Let's discuss it later.

Reply:
https://github.com/epicmaxco/vuestic-ui/issues/2216